### PR TITLE
GMOSSP amp-ad support

### PR DESCRIPTION
**Pull request by HikaruYokogoshi**
Could you merge 'GMOSSP amp-ad support' into ampproject/amphtml master branch?

We have a Google Corporate CLA.
Our CLA: GMO AD Marketing Inc.
Our company name: GMO AD Marketing Inc.

Thank you in advance

### CODE REVIEW

File: ads/gmossp.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {writeScript} from '../src/3p';
+/**
```
Comment (dvoytenko) [inline ads/gmossp.js]: nit: please add a new line here

File: ads/gmossp.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
```
Comment (dvoytenko) [inline ads/gmossp.js]: 2016 here and below

File: ads/gmossp.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+ * limitations under the License.
+ */
+
+import {writeScript} from '../src/3p';
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function gmossp(global, data) {
+  global.gmosspParam = data;
```
Comment (dvoytenko) [inline ads/gmossp.js]: Is `data-id` required? If so, it's best to assert it here via `validateDataExists`.

**Comment by HikaruYokogoshi** [comment]
@dvoytenko 
Thank you for your advice.　
I modified the source and pushed to 'HikaruYokogoshi:master' branch a minutes ago. 
Could you check my commit again?

**Comment by dvoytenko** [comment]
@HikaruYokogoshi Thanks! Merged.

**Comment by HikaruYokogoshi** [comment]
@dvoytenko Thanks a lot!

**Pull request merged**